tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -3880,6 +3880,8 @@ UnaryOperationMap() {
           {"Ceil", nvinfer1::UnaryOperation::kCEIL},
           {"Floor", nvinfer1::UnaryOperation::kFLOOR},
           {"Erf", nvinfer1::UnaryOperation::kERF},
+          {"Round", nvinfer1::UnaryOperation::kROUND},
+          {"Sign", nvinfer1::UnaryOperation::kSIGN},
```

Comment:
Are these two enums only available on certain version?
We got this error:
[third_party/tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc:3772](https://cs.corp.google.com/piper///depot/google3/third_party/tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc?l=3772&ws=tap-prod-presubmit/170409851&snapshot=2):47: error: no member named 'kROUND' in 'nvinfer1::UnaryOperation'
          {"Round", nvinfer1::UnaryOperation::kROUND},
                    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
[third_party/tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc:3773](https://cs.corp.google.com/piper///depot/google3/third_party/tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc?l=3773&ws=tap-prod-presubmit/170409851&snapshot=2):46: error: no member named 'kSIGN' in 'nvinfer1::UnaryOperation'; did you mean 'kSIN'?

Reply:
Yes, It looks like it is true. I also saw that everything is OK for our 21.12 branch, but not for 22.01.

Reply:
The two new operations were [added in TRT 8.2](https://docs.nvidia.com/deeplearning/tensorrt/release-notes/tensorrt-8.html#rel-8-2-0-EA). @drivanov pleas guard them with `#if IS_TRT_VERSION_GE(8, 2, 0, 0)`.

Reply:
@tfeher: I just added corresponding `#if IS_TRT_VERSION_GE(8, 2, 0, 0)` statements